flexget/plugins/sites/freshon.py

```diff
@@ -0,0 +1,188 @@
+            log.debug('Logging in to Freshon.tv...')
+            params = {'username': config['username'],
+                      'password': config['password'],
+                      'login': 'Do it!'}
+            urlstr = (base_url, login_fragment)
+            lsrc = task.requests.post('/'.join(urlstr), data=params)
+            if str(config['username']) in lsrc.text:
+                log.debug('Login to FreshonTV was successful')
+            elif str('Username does not exist in the userbase') in lsrc.text:
+                log.error('Invalid username or password for FreshonTV; Check your settings')
+                raise plugin.PluginError("Invalid username or Password for FreshonTV.")
+            else:
+                log.error('Login to FreshonTV was NOT successful')
+                raise plugin.PluginError("Login to FreshonTV was NOT successful")
+
+        if config['freeleech'] == 'all':
+            freeleech = 0
+        elif config['freeleech'] == 'free':
+            freeleech = 3
+        elif config['freeleech'] == 'half':
+            freeleech = 4
+
+        entries = set()
+        for search_string in entry.get('search_strings', [entry['title']]):
+            query = normalize_unicode(search_string.replace(' ', '+'))
+            query_url_fragment = urllib.quote_plus(query.encode('utf-8'))
+            str_url = (base_url, search_fragment)
+            url = '/'.join(str_url) % (query_url_fragment, config['category'], str(freeleech))
+            log.debug('Search url: %s' % url)
+            page = task.requests.get(url).content
+            soup = get_soup(page)
+            if soup.findAll(text=re.compile('Nothing found. Try again with a refined search string.')):
+                log.debug('Search returned no results')
+            else:
+                max_page_number = 0
+                # Check to see if there is more than 1 page of results
+                pager = soup.find('div', {'class': 'pager'})
+                if pager:
+                    page_links = pager.find_all('a', href=True)
+                else:
+                    page_links = []
+                if len(page_links) > 0:
+                    for lnk in page_links:
+                        link_text = lnk.text.strip()
+                        if link_text.isdigit():
+                            page_int = int(link_text)
+                            if page_int > max_page_number:
+                                max_page_number = page_int
+                else:
+                    max_page_number = 1
+
+                # limit to 10 pages
+                if max_page_number > 10:
+                    max_page_number = 10
+
+                # pages are 0 based
+                nextpage = 0
+                while (nextpage < max_page_number):
+                    if (nextpage > 0):
+                        newurl = url + '&page=' + str(nextpage)
+                        log.debug('-----> NEXT PAGE : %s' % newurl)
```

Comment:
pass string interpolation to logger, replace `%` with `,`:
` log.debug('-----> NEXT PAGE : %s', newurl)`